tests/Tokenizer/TokensTest.php

```diff
@@ -380,15 +380,6 @@ public function provideMonolithicPhpDetection()
      */
     public function testShortOpenTagMonolithicPhpDetection($source, $monolithic)
     {
-        /*
-         * short_open_tag setting is ignored by HHVM
-         * @see https://github.com/facebook/hhvm/issues/4758
-         */
-        if (!ini_get('short_open_tag') && !defined('HHVM_VERSION')) {
```

Comment:
The tests fail because of this `if` removal. You should restore it and only remove the `!defined('HHVM_VERSION')` check and the comment above.